src/main/java/pages/ListingListPage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+import features.Listing;
+import features.Search;
+
+import java.util.List;
+
+public class ListingListPage extends Page {
+    private List<Listing> listings;
+
+    // super call from page, assume every page title will be listings?
```

Comment:
If that is the way we are doing it (always passing "listing" as the title for ListingListPage) then we should discuss it since a lot of us will need to stick to this format.

Reply:
do you want like the website name on there instead?

Reply:
@MarkianV when would we want it to be something other than listings? i don't see a problem with hardcoding it into every instance of the listing list page